doc/libghdl/ASTTraversing.rst

```diff
@@ -0,0 +1,12 @@
+.. _LIB:ASTTraversing:
+
+AST Traversing
+##############
+
+If someone wants to use `libghdl`, then he probably wants to know how to traverse the AST to get the information about nodes of design units.
+It would be really nice to provide an example showing how to traverse the tree.
```

Comment:
Have a look at `vhdl_langserver/symbols.py` to see how symbols are gathered from a vhdl file. Or `references.py`.